R/convertToPhyloseq.R

```diff
@@ -1,22 +1,19 @@
-#' Create a phyloseq object from a TreeSummarizedExperiment object
-#'
+#' Create a phyloseq object from a \code{TreeSummarizedExperiment} object
+#'  
+#' @inheritParams convertFromBIOM
+#' 
 #' @param x a 
 #'   \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-class]{TreeSummarizedExperiment}}
 #'   object
 #'
-#' @param assay.type A single character value for selecting the
-#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{assay}} to be
-#'   included in the phyloseq object that is created. 
-#'   (By default: \code{assay.type = "counts"})
+#' @param assay.type \code{Character scalar}. Specifies the name of assay 
```

Comment:
"used in calculation". This function does not calculate. See the removed description